mono/mini/local-propagation.c

```diff
@@ -841,7 +841,7 @@ static gboolean
 mono_is_simd_accessor (MonoInst *ins)
 {
 	switch (ins->opcode) {
-#ifdef MONO_ARCH_SIMD_INTRINSICS
+#if defined(MONO_ARCH_SIMD_INTRINSICS) && !defined(TARGET_ARM64)
```

Comment:
Why is this needed ?

Reply:
these ops are defined for x86 only in mini-ops but yeah I'll share them with arm and remove this if-def